engine/src/main/scala/cromwell/engine/backend/Backend.scala

```diff
@@ -140,12 +147,10 @@ trait Backend {
 
   def executionInfoKeys: List[String]
 
-  def callRootPathWithBaseRoot(jobDescriptor: BackendCallJobDescriptor, baseRoot: String): Path = {
-    jobDescriptor.key.callRootPathWithBaseRoot(jobDescriptor.workflowDescriptor, baseRoot)
-  }
+  def callRootPathWithBaseRoot(key: CallKey, descriptor: WorkflowDescriptor, baseRoot: String): Path = Backend.callRootPathWithBaseRoot(key, descriptor, baseRoot)
```

Comment:
This is a left over from my failed attempt to decouple the various keys, I'll revert it